exonum-java-binding-core/rust/integration_tests/Cargo.toml

```diff
@@ -11,10 +11,8 @@ crate-type = ["dylib"]
 [features]
 
 [dependencies]
-futures = "0.1.14"
+futures = "0.1"
 java_bindings = { path = "..", features = ["invocation"] }
-exonum-testkit = "0.9"
-lazy_static = "1.2.0"
-rand = "0.6"
-serde = "1.0"
-serde_derive = "1.0"
+exonum-testkit = { git = "https://github.com/exonum/exonum" }
+lazy_static = "1.1.0"
```

Comment:
Why are the deps downgraded (lazy & rand)?

Reply:
🤷‍♂️ Will fix.